cmd/shell.go

```diff
@@ -0,0 +1,109 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"syscall"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// sigCh is called to shutdown the server.
+func runShell(mountpath string, sigCh chan os.Signal) exitCode {
+	cleanup := func(msgs ...interface{}) exitCode {
+		log.Print(msgs...)
+		sigCh <- syscall.SIGINT
+		return exitCode{1}
+	}
+
+	// Run the default system shell.
+	sh := os.Getenv("SHELL")
+	if sh == "" {
+		sh = "/bin/sh"
+	}
+
+	// Add wash to PATH, set PS1. These are useful fallbacks in case PATH and prompt aren't setup.
+	washPath, err := os.Executable()
+	if err != nil {
+		return cleanup(err)
+	}
+	washDir := filepath.Dir(washPath)
+	pathEnv := os.Getenv("PATH")
+	if err := os.Setenv("PATH", pathEnv+string(os.PathListSeparator)+washDir); err != nil {
```

Comment:
Are we still planning on aliasing the Wash commands? It's a bit annoying having a `wash$` prompt but still having to type `wash ls`, `wash find`, etc. to do Wash-specific stuff.

Reply:
Probably. I don't think LS is useful enough to be the default yet though.